sql/moz-fx-data-shared-prod/accounts_backend_derived/monitoring_db_recovery_phones_counts_v1/query.sql

```diff
@@ -0,0 +1,11 @@
+SELECT
+  @as_of_date AS as_of_date,
+  SAFE.JSON_VALUE(lookupData, '$.countryCode') AS country_code,
+  COUNT(uid) AS total_rows
+FROM
+  `moz-fx-data-shared-prod.accounts_db_external.fxa_recovery_phones_v1` FOR SYSTEM_TIME AS OF TIMESTAMP(
+    @as_of_date + 1,
+    'UTC'
+  )
```

Comment:
The CI checks have reminded me that since this ETL now uses time travel it needs to be added to the dry-run skip list [like `monitoring_db_counts_v1` is](https://github.com/mozilla/bigquery-etl/blob/95320f6965e30d530f07d268e91bf50b99611425/bqetl_project.yaml#L47-L48) because the current dry-run logic tries to run it for a hard-coded historical date which is outside of the 7-day time travel window, causing it to [fail](https://app.circleci.com/pipelines/github/mozilla/bigquery-etl/46918/workflows/9efd3f51-3970-4255-9a31-f5ba25a72987/jobs/548732):
> Invalid time travel timestamp 1546387200000 for table moz-fx-data-shared-prod:accounts_db_external.fxa_recovery_phones_v1@1546387200000. Cannot read before 1741631681238

Reply:
so many moving parts in this thing!  new patch adds it